client/landing/stepper/declarative-flow/tailored-ecommerce-flow.ts

```diff
@@ -48,16 +48,16 @@ function getPlanFromRecurType( recurType: string ) {
 
 const ecommerceFlow: Flow = {
 	name: ECOMMERCE_FLOW,
-	useSteps() {
-		const recurType = useSelect(
+	isSignupFlow: true,
+	useSignupStartEventProps() {
+		const recur = useSelect(
 			( select ) => ( select( ONBOARD_STORE ) as OnboardSelect ).getEcommerceFlowRecurType(),
 			[]
 		);
 
-		useEffect( () => {
```

Comment:
If we're not going to call `useSignupStartEventProps()` in this PR, might it be cleaner to leave the secondary event logging so we at least keep getting the events with `recur` defined? Otherwise we won't be able to see which flow the user actually went through for pricing etc.

Reply:
I've gone ahead and removed `useSignupStartEventProps()` from this PR entirely, and left things as they were in the original implementation, duplicate event and all. The change that does this is 5b604fec115b62601e6b795b41c9333ddb9bf927.